tests/test_engines.py

```diff
@@ -68,3 +69,20 @@ def test_simulate_nonexistent_param(self):
             engine.simulate(**good_kwargs)
             with self.assertRaises(TypeError):
                 engine.simulate(**bad_kwargs)
+
+    def test_required_params(self):
+        species = stdpopsim.get_species("HomSap")
+        model = species.get_demographic_model("AshkSub_7G19")
+        contig = (species.get_contig("chr1"),)
+        for engine in stdpopsim.all_engines():
+            with self.assertRaises(TypeError):
+                engine.simulate(model, contig)
+
+    def test_msprime_kwargs(self):
+        species = stdpopsim.get_species("HomSap")
+        model = species.get_demographic_model("AshkSub_7G19")
+        contig = species.get_contig("chr22", length_multiplier=0.01)
+        samples = model.get_samples(10)
+        engine = stdpopsim.get_engine("msprime")
+        replicates = engine.simulate(model, contig, samples, num_replicates=2)
```

Comment:
``num_replicates`` might not be a great example here as this could get implemented in the main API some day. Something that's msprime specific like ``record_full_arg`` is probably better, and you can test for the existance of recomb nodes.

Good to set the seed as well to make this deterministing.